### PR TITLE
New cards for B decays

### PR DESCRIPTION
**Pull request by jnovoafe**
Add 16 new cards for several B decays, including neutral modes and charged lepton modes.

### CODE REVIEW

File: FCCee/Generator/EvtGen/Bd2KsMuMu.dec [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Alias   Bs_SIGNAL   B_s0
```
Comment (zuoxunwu) [inline FCCee/Generator/EvtGen/Bd2KsMuMu.dec:1]: Same comment for `Bd` vs `Bs` decays.

File: FCCee/Generator/EvtGen/Bd2KsNuNu.dec [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Alias   Bs_SIGNAL   B_s0
```
Comment (zuoxunwu) [inline FCCee/Generator/EvtGen/Bd2KsNuNu.dec:1]: Same comment for `Bd` vs `Bs` decays

File: FCCee/Generator/EvtGen/Bd2KsTauTau.dec [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Tauola steering options
+Define TauolaCurrentOption 0
+Define TauolaBR1 1.0
+#
+Alias         Mytau+     tau+
+Alias         Mytau-     tau-
+ChargeConj    Mytau+     Mytau-
+
+Alias   Bs_SIGNAL   B_s0
```
Comment (zuoxunwu) [inline FCCee/Generator/EvtGen/Bd2KsTauTau.dec:9]: Same comment for `Bs` vs `Bs` decays

File: FCCee/Generator/EvtGen/Bd2KsEE.dec [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Alias   Bs_SIGNAL   B_s0
```
[the remark this answers is not in the document]
Comment (jnovoafe) [reply FCCee/Generator/EvtGen/Bd2KsEE.dec]: Hi @zuoxunwu, thank you for reviewing these cards. Indeed those four decays that you mentioned should use Bd, not Bs. I have just updated the pull request with the corrected decays.

File: FCCee/Generator/EvtGen/Lb2LbEE.dec [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Alias   Lbbar_SIGNAL    anti-Lambda_b0
+ChargeConj Lb_SIGNAL  Lbbar_SIGNAL
+
+Alias      MyLambda0      Lambda0
+Alias      Myanti-Lambda0 anti-Lambda0
+ChargeConj MyLambda0      Myanti-Lambda0
+
+#
+Decay Lb_SIGNAL
+  1.000        MyLambda0    e+    e-        BTOSLLBALL;
```
Comment (zuoxunwu) [inline FCCee/Generator/EvtGen/Lb2LbEE.dec:11]: Got error during generation 
```
EvtGen:BTOSLLBALL did not get the correct parent spin
Lb_SIGNAL -> MyLambda0 e+ e-  (BTOSLLBALL)
```
Could you double check the implementation of this model?

File: FCCee/Generator/EvtGen/Lb2LbMuMu.dec [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+Alias   Lbbar_SIGNAL    anti-Lambda_b0
+ChargeConj Lb_SIGNAL  Lbbar_SIGNAL
+
+Alias      MyLambda0      Lambda0
+Alias      Myanti-Lambda0 anti-Lambda0
+ChargeConj MyLambda0      Myanti-Lambda0
+
+#
+Decay Lb_SIGNAL
+  1.000        MyLambda0    mu+    mu-         BTOSLLBALL;
```
Comment (zuoxunwu) [inline FCCee/Generator/EvtGen/Lb2LbMuMu.dec:11]: Got error during generation
```
EvtGen:BTOSLLBALL did not get the correct parent spin
Lb_SIGNAL -> MyLambda0 mu+ mu-  (BTOSLLBALL)
```
Could you double check the implementation of this model?

File: FCCee/Generator/EvtGen/Lb2LbTauTau.dec [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+Alias   Lbbar_SIGNAL    anti-Lambda_b0
+ChargeConj Lb_SIGNAL  Lbbar_SIGNAL
+
+Alias      MyLambda0      Lambda0
+Alias      Myanti-Lambda0 anti-Lambda0
+ChargeConj MyLambda0      Myanti-Lambda0
+
+#
+Decay Lb_SIGNAL
+  1.000        MyLambda0    Mytau+    Mytau-         BTOSLLBALL;
```
Comment (zuoxunwu) [inline FCCee/Generator/EvtGen/Lb2LbTauTau.dec:18]: Got error during generation
```
EvtGen:BTOSLLBALL did not get the correct parent spin
Lb_SIGNAL -> MyLambda0 Mytau+ Mytau-  (BTOSLLBALL)
```
Could you double check the implementation of this model?

**Comment by zuoxunwu** [comment]
Dear Julio,
Thanks for the PR and for already replying to my comments:)
I have added a few points I found during tests. I think once they are addressed, the PR is good to be merged.

I have one additional suggestion, not really a comment:
In cards like [Lb2LbMuMu.dec](https://github.com/HEP-FCC/FCC-config/pull/152/files/40f329ba1ff4938b0c204ce7bdf52f3393fa6aea#diff-07581f13e9295f45b8d25eac8cdc7ce3138a31da15c65eef5748ffc325e71787) , it is a bit misleading to use `Lb` for both `Lambda_b0` and `Lambda_0`. I would suggest to keep `Lb` for `Lambda_b0`, but either `L` or `Lam` for `Lambda_0`. 
I am not sure about the convention for this in LHCb. Maybe we can follow their convention? Or maybe @EmanuelPerez has a better idea.
There is one existing card [Lb2LbNuNu.dec](https://github.com/HEP-FCC/FCC-config/blob/winter2023/FCCee/Generator/EvtGen/Lb2LbNuNu.dec) that has the same issue. (I would rename that card once we have a decision.)

**Comment by jnovoafe** [comment]
Dear Xunwu, thank you for checking all the files. Indeed the naming for those Lambda decays was ambiguous. I have changed the cards and now Lb stands for Lambda_b and L for Lambda_0, which is the LHCb standard.

Also, it looks like BTOSLLBALL will not work for Lambda decays because of its spin. After asking the LHCb simulation team, we were advised to use RareLbToLll - it has been used for a few LHCb cards before. Hopefully this model will work for these decays too. Thank you again for your help!

**Comment by zuoxunwu** [comment]
Dear Julio,
Thanks for fixing it. The `RareLbToLll` works indeed.
Now everything looks good. I will merge this PR and move on to production.
Best,
Xunwu

**Pull request merged**